lib/galaxy/managers/workflows.py

```diff
@@ -1584,12 +1587,12 @@ def callback(input, prefixed_name, **kwargs):
             steps[step.order_index] = step_dict
         return data
 
-    def _workflow_to_dict_instance(self, stored, workflow, legacy=True):
+    def _workflow_to_dict_instance(self, stored, workflow, legacy=True, url_builder=None):
```

Comment:
Maybe have `trans` as first parameter instead, as the other similar methods?

Reply:
I tried to explicitly avoid passing `trans` and be more specific about what this function really needs from it, but I guess it's not worth the inconsistency with the rest of the functions, and the ship of "not using trans everywhere" has sailed already :laughing: 

I will do it for consistency.